source/particle/integrator/interface.cc

```diff
@@ -68,6 +68,15 @@ namespace aspect
 
 
 
+      template <int dim>
+      std::array<bool, 3>
+      Interface<dim>::required_solution_vectors() const
+      {
+        return {{false, true, true}};
+      }
```

Comment:
Why is this the right default? I would have expected that you err on the safe side and return `false, false, false`. But perhaps it would make sense to just not implement this function at all (i.e., mark it as `=0` in the `.h` file)?